apiary.apib

```diff
@@ -1210,12 +1210,13 @@ An optional expiration (seconds) and reason for joining can be added to the requ
 + Attributes
 
     + email: martin@keboola.com (string, required) - Email of an invited user
-    + role: admin (enum[string]) - User role in the project. Roles `admin` and `guest` are allowed (default `admin`).
+    + role: admin (enum[string]) - User role in the project. Roles `admin`, `guest` and `readOnly` are allowed (default `admin`).
     
         + Default: `admin`
         
         + Members
             + `guest`
+            + `readOnly`
```

Comment:
zkontroloval jsem, že tady být admin uvedený nemusí a nepřidává to žádný warning